src/test/kotlin/logic/use_cases/log/SaveLogUseCaseTest.kt

```diff
@@ -22,7 +22,7 @@ class SaveLogUseCaseTest{
    saveLogUseCase = SaveLogUseCase(logRepository)
   }
  @Test
- fun `should call addLog on repository when saveLog is invoked`() {
+ fun `should save log entry in repository when called`() {
```

Comment:
We should clearly specify which thing is called, so we may rename it to:

```suggestion
 fun `should save log entry in repository when addLog called`() {
```

Reply:
Thank you very much for reminding me about the naming